graphqlapi/src/test/java/io/stargate/graphql/schema/GraphQlTestBase.java

```diff
@@ -31,8 +31,8 @@
 import io.stargate.db.query.builder.AbstractBound;
 import io.stargate.db.query.builder.QueryBuilder;
 import io.stargate.db.schema.Schema;
-import io.stargate.graphql.web.HttpAwareContext;
-import io.stargate.graphql.web.HttpAwareContext.BatchContext;
+import io.stargate.graphql.web.StargateGraphqlContext;
```

Comment:
I think that we need a change in c2 (and possibility CNDB) because those changes are not backward compatible and those projects might use this API.

Reply:
I'll double check, but I'm pretty sure C2 and CNDB do not reference that API.